kasa/tests/conftest.py

```diff
@@ -46,7 +46,7 @@
 BULBS_SMART_VARIABLE_TEMP = {"L530E"}
 BULBS_SMART_COLOR = {"L530E"}
 BULBS_SMART_LIGHT_STRIP: Set[str] = set()
-BULBS_SMART_DIMMABLE: Set[str] = set()
+BULBS_SMART_DIMMABLE = {"KS225"}
```

Comment:
Maybe this should go in DIMMABLE_SMART as it doesn’t seem to have any other bulb functions?

Reply:
We may need to change this in the future, when we get some more experience and reports on other "SMART" devices. We have `SmartDimmer` for IOT devices, which sort of implements the switch interface with added brightness controls, so we may want to introduce `TapoDimmer` (or something) later on.

I think this is fine for the time being though, I suppose the interfaces are more similar across SMART devices than they were for IOT which helps us :-)

Reply:
I worked on [adding taposwitch.py](https://github.com/gimpy88/python-kasa/tree/add_taposwitchl) and used my KS205 and KS225 devices with it. It's presently a little hacky, but it works and I think it looks cleaner. Thoughts?

Reply:
#601 is now merged.

On separating the code into taposwitch, I think we should rather wait until we finalize the structure of the "tapo" package (which is a misnomer given it's not tapo only :shrug:). So if it's working just fine as "TapoBulb", it's better to keep it as-is for the time being :-)

Reply:
Sounds good!